spring-cloud-dataflow-rest-client/src/main/java/org/springframework/cloud/dataflow/rest/client/TaskTemplate.java

```diff
@@ -47,6 +49,8 @@
 
 	private static final String EXECUTIONS_RELATION = "tasks/executions";
 
+	private static final String EXECUTIONS_CURRENT_RELATION = "tasks/executions/current";
```

Comment:
This should work, but when looking at the view command i.e. https://github.com/spring-cloud/spring-cloud-dataflow/blob/master/spring-cloud-dataflow-server-core/src/main/java/org/springframework/cloud/dataflow/server/controller/TaskExecutionController.java#L176.  Their urls are very similar.   Need to see if this is ok style wise with our RESTful API gurus.

Reply:
Need to update the copyright date from 2017 to 2018

Reply:
Chatted with the RESTful expert (@mminella ;-) )  and they said we are good to go with the endpoint.  But i did like his recommendation to rename it from `/tasks/executions/current` to `/tasks/executions/currentcount`.

Reply:
I considered something like that. But it also gets the max concurrent so I
thought something more generic is appropriate.